images/s6_assets/galaxy_ng_tests.sh

```diff
@@ -3,30 +3,30 @@ set -euo pipefail
 
 SERVER="localhost"
 WEB_PORT="8080"
+scheme=${1:-http}
 
 pip3 install "ansible<2.13.2"
 
-BASE_ADDR="http://$SERVER:$WEB_PORT"
-echo $BASE_ADDR
```

Comment:
This line would be nice to have for debugging, albeit with a better print statement:
```suggestion
echo "BASE_ADDR: $BASE_ADDR"
```

Reply:
I deleted that line since it is a duplicate, the line right below prints the same thing with the better message.